src/pages/[id].page.jsx

```diff
@@ -32,17 +32,29 @@ export default function Details({ details }) {
 
   // dynamic imports need to be asynchronous
   useEffect(() => {
-    const updateCodeBlocks = async () => {
-      const html = await import(`../code-examples/${slug}/html.js`);
-      setHtmlCode(html.default);
+    const updateSandpackExamples = async () => {
+      try {
+        const html = await import(`../code-examples/${slug}/html.js`);
+        setHtmlCode(html.default);
 
-      const css = await import(`../code-examples/${slug}/css.js`);
-      setCssCode(css.default);
+        const css = await import(`../code-examples/${slug}/css.js`);
+        setCssCode(css.default);
 
-      const javascript = await import(`../code-examples/${slug}/javascript.js`);
-      setJavascriptCode(javascript.default);
+        const javascript = await import(
+          `../code-examples/${slug}/javascript.js`
+        );
+        setJavascriptCode(javascript.default);
+
+        // console.log('all code is here');
+      } catch (e) {
+        if (htmlCode || cssCode || javascriptCode === '') {
```

Comment:
This is good but we will want to update the syntax of this ```if```

```
 if (htmlCode === ' ' || cssCode ==== ' ' || javascriptCode === ' ') {...
```

Reply:
We need to check each value for an empty string or it will only checking if the value is  ```defined || null```

Reply:
Got it, fixed here: https://github.com/sparkbox/accessible-components/pull/132/commits/7393af4b682699ad4d5484f899f50772073d0d24